recipes/julia/build.sh

```diff
@@ -1,3 +1,2 @@
 #!/bin/sh
-make
-env PREFIX=$PREFIX prefix=$PREFIX make install
+cp -r * $PREFIX
```

Comment:
Hmm how about this way. It seems like LICENSE.md is the only unusual file in the tar.